includes/theme-navigation.php

```diff
@@ -134,13 +135,13 @@ public function bu_filter_anchor_attrs( $attrs ) {
 	 * @return array
 	 */
 	public function bu_navigation_filter_item_attrs( $item_classes ) {
-		if ( in_array( 'current_page_item', $item_classes, true ) ) {
```

Comment:
I noticed the same things, and was about to push the following for this function:
```
$remove_classes = array( 'page_item', 'current_page_item', 'current_page_parent' );

if ( in_array( 'current_page_item', $item_classes, true ) ) {
	$item_classes[] = 'active';
}

return array_diff( $item_classes, $remove_classes );
```
Thoughts?

And thanks for remembering the BU stuff, btw. I feel lazy for having missed it! :)